### PR TITLE
Patch templates

### PR DESCRIPTION
**Pull request by jmillanacosta**
- Add to templates some terms that had been added to the glossary page directly
- Document templates workflow

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -353,17 +353,17 @@ Experienced GitHub users are encouraged to submit pull requests. More informatio
...
+    <td property="ncit:C42610">A software entity that uses a language model to autonomously perform specific tasks by interpreting prompts, interacting with data or systems, and generating results.</td>
```
Comment (egonw) [inline index.md:357]: Something seems to go wrong here: the description seems to end up in the Markdown as the acronym

File: index.md [hunk 2]
```diff
@@ -543,12 +543,12 @@ Experienced GitHub users are encouraged to submit pull requests. More informatio
...
+    <td><span property="rdfs:label">Pompt</span> <a href="https://vhp4safety.github.io/glossary#VHP0000133"><svg xmlns="http://www.w3.org/2000/svg" height="12" width="15" viewBox="0 0 640 512"><path d="M579.8 267.7c56.5-56.5 56.5-148 0-204.5c-50-50-128.8-56.5-186.3-15.4l-1.6 1.1c-14.4 10.3-17.7 30.3-7.4 44.6s30.3 17.7 44.6 7.4l1.6-1.1c32.1-22.9 76-19.3 103.8 8.6c31.5 31.5 31.5 82.5 0 114L422.3 334.8c-31.5 31.5-82.5 31.5-114 0c-27.9-27.9-31.5-71.8-8.6-103.8l1.1-1.6c10.3-14.4 6.9-34.4-7.4-44.6s-34.4-6.9-44.6 7.4l-1.1 1.6C206.5 251.2 213 330 263 380c56.5 56.5 148 56.5 204.5 0L579.8 267.7zM60.2 244.3c-56.5 56.5-56.5 148 0 204.5c50 50 128.8 56.5 186.3 15.4l1.6-1.1c14.4 10.3 17.7-30.3 7.4-44.6s-30.3-17.7-44.6-7.4l-1.6 1.1c-32.1 22.9-76 19.3-103.8-8.6C74 372 74 321 105.5 289.5L217.7 177.2c31.5-31.5 82.5-31.5 114 0c27.9 27.9 31.5 71.8 8.6 103.9l-1.1 1.6c-10.3 14.4-6.9 34.4 7.4 44.6s34.4-6.9 44.6-7.4l1.1-1.6C433.5 260.8 427 182 377 132c-56.5-56.5-148-56.5-204.5 0L60.2 244.3z"/></svg></a></td>
```
Comment (egonw) [inline index.md:546]: Small typo here.

**Review by egonw — CHANGES_REQUESTED** [review]
This looks great!

File: index.md (hunk 1 above)
Comment (jmillanacosta) [reply index.md]: Fixed with [d6df6ea](https://github.com/VHP4Safety/glossary/pull/91/commits/d6df6ea206aeb3d31cc152f48109b9b7ef2f8a9e), typo in the template

File: index.md (hunk 2 above)
Comment (jmillanacosta) [reply index.md]: Fixed with [d6df6ea](https://github.com/VHP4Safety/glossary/pull/91/commits/d6df6ea206aeb3d31cc152f48109b9b7ef2f8a9e), typo in the template

**Comment by jmillanacosta** [comment]
Ready @egonw @ozancinar

**Pull request merged**